src/schema/feeds.ts

```diff
@@ -894,6 +894,11 @@ export const typeDefs = /* GraphQL */ `
     Update user's feed advanced settings
     """
     updateFeedAdvancedSettings(
```

Comment:
We need to add feedPlus here.

Reply:
yes, gonna update after your PR is merged